core/libraries/messages/messenger/EE_Email_messenger.class.php

```diff
@@ -609,8 +609,7 @@ protected function _body($preview = false)
          */
         if (apply_filters('FHEE__EE_Email_messenger__apply_CSSInliner ', true, $preview)) {
             // require CssToInlineStyles library and its dependencies via composer autoloader
-            require_once EE_THIRD_PARTY . 'cssinliner/vendor/autoload.php';
-
+            require_once EE_VENDOR . 'autoload.php';
```

Comment:
I realized that the `autoload.php` file is not loaded by default so added this line to prevent issues. We can require this file in a more general place and then remove it from this part.